nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryAPI.java

```diff
@@ -3743,6 +3743,61 @@ public void testImport() {
         assertFalse(doc.isProxy());
     }
 
+    @Test
+    public void testImportDuplicateIdSameSession() {
+        DocumentModel doc = session.createDocumentModel("/", "doc", "File");
+        doc = session.createDocument(doc);
+        session.save();
+        nextTransaction();
```

Comment:
could/should these 2 lines be replaced by
```
transactionalFeature.nextTransaction()
```
Though it's probably for backport purpose, don't we want to benefit from it on master?

Reply:
For tests I don't always bother to try to find the optimal sequence if it's not important to the test. Also I prefer to do the `save` explicitly because in some of the tests it's at save time that a failure occurs, and hiding it inside the commit's flush wouldn't help make things obvious. Finally `save` + `nextTransaction` is already used in a number of other places in this same file.

Reply:
OK, we really need to write these unit testing guidelines as such statements aren't always obvious.